### DISCUSSION TITLE
KATA 2

### DISCUSSION THREAD

**Discussion by PaulinaTorrecillas** [LaunchX-InnovaccionVirtual/CursoIntroPython#62] [Q&A] [answered]
No puedo avanzar desde el instalador de python3 -m venv envo bienpython -m venv env
![Erro Python](https://user-images.githubusercontent.com/87047080/153322513-b7d3c26d-d265-4e90-8087-53a85a98656d.PNG)
Si alguien puede ayudarme se lo agradecería mucho!!

**Comment by OscarPedroza** [comment]
Hola, intenta usar solo el comando python en lugar de python3, quedaría algo asi: `python -m venv env`

**Reply by PaulinaTorrecillas** [reply]
Hola ya intente pero no entra

**Reply by GeovanniBlanco** [reply]
Hola puedes intentar con py -m venv env     
a mi me funciono

**Reply by JuanMedina22** [reply]
@GeovanniBlanco me salvaste la vida, viejo

**Reply by ghost** [reply]
Hola, usé python en lugar de python3, pero no se genero ninguna carpeta bin

**Reply by CamilaMartinezVilla** [reply]
@JulianHernandezJara  Tengo el mismo problema ¿alguien que sepa que pasa?
Lei algo en la documentaicon ofical de python, pero no lo comprendo del todo ): y no se si tenga que ver:
"If you are using Python 3, then you should already have the [venv](https://docs.python.org/3/library/venv.html) module from the standard library installed."

**Comment by MSuarez0** [comment]
lo estas ejecutando desde VC code o desde la terminal(cmd , cli, terminal) de tu equipo? y teniendo en cuenta que ya tienes intalado la versión mas nueva de python?

**Reply by PaulinaTorrecillas** [reply]
Hola desde la terminal cmd si ya tengo instalada la siguiente version
![image](https://user-images.githubusercontent.com/87047080/153327285-1d1054f5-0ba8-4caa-9f21-1afac77c5122.png)

**Comment by ViktorVizcarra** [comment]
> No puedo avanzar desde el instalador de python3 -m venv envo bienpython -m venv env ![Erro Python](https://user-images.githubusercontent.com/87047080/153322513-b7d3c26d-d265-4e90-8087-53a85a98656d.PNG) Si alguien puede ayudarme se lo agradecería mucho!!

creo que tiene que ejecutar ese comando pero dese la ruta donde se instalo python en el equipo, tambien debiste elejir la opcion add tho path al istalar python.

**Comment by KevinChavarin** [comment]
> No puedo avanzar desde el instalador de python3 -m venv envo bienpython -m venv env ![Erro Python](https://user-images.githubusercontent.com/87047080/153322513-b7d3c26d-d265-4e90-8087-53a85a98656d.PNG) Si alguien puede ayudarme se lo agradecería mucho!!

Instala Python 3.10 desde la Microsoft Store, eso soluciona el problema

**Reply by Jhony8691** [reply]
quitale el 3 a python, que quede como python -m venv env

![image](https://user-images.githubusercontent.com/83784004/153543014-ca7f193a-f475-40a5-acca-ec67fe85ab53.png)

**Comment by PlayerOneV** [comment]
La consola te esta señalando que no encuentra Python, se me ocurren dos motivos:
1.- No haz instalado Python en tu computadora.
2.- Instalaste Python pero no se encuentra entre tus variables globales por lo que no puedes usar sus comandos desde cualquier dirección(carpeta) del equipo.
Para saber si ya tienes Python abre "cmd" y escribe el comando "python --version"
![image](https://user-images.githubusercontent.com/75816087/153326414-fa7c95c0-30c8-49b9-b0f6-e8e4cfe23101.png)
Si no la tienes puedes buscar y descargar el .exe y recuerda dejar seleccionada la casilla de añadir Python al Path.

**Reply by PlayerOneV** [reply]
Intenta hacerlo desde otra carpeta que no sea la de OneDrive

**Comment by DiegoHC06** [comment]
lo que voy entendiendo es que el comando Python no lo encuentra dicho comando se utilizaba en otros sistemas,  eso se debe a que las ultimas versiones es el método correcto para ejecutar programas u otros comandos. intenta intenta con solo py. en caso que te error de nuevo es que no esta instalado o es una mala configuración, para que revises una configuración adicional, puede escribir en el buscador de Windows alias -> clic el resultado -> luego buscas las Python y por ultimo si están desactivadas, actívalas. listo

**Comment by AldahirGarcia98** [comment]
Busca la opción "**_Administrar alias de ejecución de aplicaciones_**"

![image](https://user-images.githubusercontent.com/94207938/153329243-83837cff-d9eb-4b32-8ba8-06c60f94f6bd.png)

Ya que estés ahí, **_deshabilita/apaga las opciones que tengan python.exe_** (en mi caso son las ultimas dos).
Así resolví ese problema, ojala te funcione c:

![image](https://user-images.githubusercontent.com/94207938/153329453-b630cfe7-c0bf-4ba4-92b8-fc4b60dbaf28.png)

**Comment by PaulinaTorrecillas** [comment, marked as answer]
Ya pude muchas gracias a todos!! @DiegoHC06 @PlayerOneV @OscarPedroza @AldahirGarcia98 @KevinChavarin @ViktorVizcarra @MSuarez0

**Reply by TorresTorresMJ** [reply]
Hola! A final qué te funciono? Porque me pasa exactamente lo mismo y ya intente con las variantes de py, pyhton, y desactivando las alias que mencionan, ejecutandolo tanto en terminal (Python was not fpund) como en la consola (invlaid syntaxis en venv)

**Reply by PaulinaTorrecillas** [reply]
@TorresTorresMJ Claro te mando un video que mire en YouTube y con ese me funciono muy bien!! https://www.youtube.com/watch?v=N9PdRkXOQ4w&t=476s
Realmente es muy parecido a la actividad espero te ayude!!

**Reply by Dfredude** [reply]
Por favor marca la pregunta como respondida o archívala.

**Comment by carlos-gmx3** [comment]
Python viene en 2 sabores, el que instalas desde la Microsoft Store o el que puedes descargar en python.org
Para este segúndo necesitas indicarle a Windows la ruta donde está el ejecutable para que sea conciente de que dispone de una aplicación que puede interpretar los programas escritos con Python
checa el siguiente enlace https://tecnoloco.istocks.club/como-agregar-python-a-la-variable-path-de-windows-wiki-util/2020-10-14/
Despu'es de agregar la ruta de Python en las variables de entorno de berías ser capaz de ejecutar el siguiente comando
python --version
Yo hice el ejercicio a traves de la terminal de Windows o cmd
Con el uso del comando "cd" que representa change directoey o cambiar directorio te desplazas a la carpeta donde estes realizando el curso. Una vez ahí puedes continuar con lo que la kata 2 te solicita
Puedes practicar primero con lo descrito en la información del modulo previo a la kata.

**Reply by PaulinaTorrecillas** [reply]
Muchas gracias @carlos-gmx3 realmente tienes razón en lo que dices!! pero ya quedo lista la Kata 2

**Comment by doguedogue** [comment]
Genial, no olvides marcar como contestada si así lo consideras!

![image](https://user-images.githubusercontent.com/23409026/153478786-4193180a-5dd4-4270-b45a-66ce95b026e1.png)

**Reply by PaulinaTorrecillas** [reply]
@doguedogue De acuerdo listo!!
